src/context/create.ts

```diff
@@ -1,15 +1,23 @@
 import { Config } from "../config/types";
 import { createSchemaContext } from "./schema";
 import { createDatabaseContext } from "./db";
-import { createTheGraphContext } from "./subgraphProvider";
+import { createTheGraphContext, GraphQlContext } from "./subgraphProvider";
```

Comment:
```suggestion
import { createSubgraphContext, GraphQlContext } from "./subgraphProvider";
```
I know that this name was before but I would change it to createSubgraphContext

Reply:
Done!